dockerfiles/itest/itest/run_itest.sh

```diff
@@ -5,8 +5,13 @@ set -e
 echo "installing paasta-tools (dependency of nerve-tools.)"
 . /etc/lsb-release
 PAASTA_VERSION=0.145.0
-PAASTA_DEB_NAME=paasta-tools_${PAASTA_VERSION}.${DISTRIB_CODENAME}1_amd64.deb
-wget "https://github.com/Yelp/paasta/releases/download/v${PAASTA_VERSION}/${PAASTA_DEB_NAME}"
+if [[ "$DISTRIB_CODENAME" == "xenial" ]]; then
```

Comment:
might be nice to include a comment about this, but `git blame` is probably good enough given that we can hopefully delete all this xenial 💩 soon :p